pkg/storage/store.go

```diff
@@ -4,6 +4,10 @@ import (
 	"context"
 	"flag"
 
+	"github.com/grafana/loki/pkg/storage/stores"
```

Comment:
Something is wrong with the formatting of those 2 new packages. They should be with everything loki related.

If you're using vs-code I think there is a way to configure it so that it does it automatically. https://gist.github.com/cyriltovena/d52bf9ae05c371ea0c8018d48d15d6bf might help you.